package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "bundler": "yarn --cwd packages/bundler bundler",
-    "runop": "yarn --cwd packages/bundler runop",
+    "runop": "yarn --cwd packages/bundler runop --deployFactory",
```

Comment:
the reason I didn't put it there in the first place, is that you can run "runop" againts a real network, and you might not want automatically to deploy the factory (e.g. you specify a non-testnet network)
an automatic "deploy" is possible, but only for local networks.

Reply:
auto-deploy only for 1337, 31337